web/virtual_lab/apps.py

```
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)